app/Http/Controllers/EmployeeController.php

```diff
@@ -11,52 +11,88 @@
 use Illuminate\Validation\Rule;
 use Illuminate\Validation\ValidationException;
 
+use function PHPUnit\Framework\isNull;
+
 class EmployeeController extends Controller
 {
     public function index(Request $request){
         $companies = Company::withTrashed()->get();
-        return view('admin' , [ 'companies' => $companies]);
+        $positions = Employee::pluck('position');
+        return view('admin' , [ 'companies' => $companies , 'positions' => $positions ]);
     }
 
     public function draw(Request $request){
 
-          //SERVER SIDE RENDERING HANDLE FOR data table
-          $search = $request->query('search')['value'];
-          $draw = $request->query('draw', 1);
-          $start = $request->query('start', 0);
-          $length = $request->query('length', 10);
-          $totalEmployees =   Employee::count();
-
-          $employees = Employee::with(['bankAccount' , 'company' => function($query) {
-                                 $query->withTrashed();
-                                }])
-                              ->where('name' , 'like' , "%".$search."%")
-                              ->orWhere('position' ,'like' , "%".$search."%")
-                              ->orWhere('email', 'like' , "%".$search."%")
-                              ->orWhere('address' ,'like' , "%".$search."%")
-                              ->orWhere('dob' ,'like' , "%".$search."%")
-                              ->orWhere('phone' ,'like' , "%".$search."%")
-                              ->orWhereHas('bankAccount',
-                              function ($q) use ($search) {
-                                  $q->where('account_no', 'like', "%".$search."%")->select('branch'); })
-                              ->orWhereHas('company', function ($q) use ($search) {
-                                  $q->withTrashed()
-                                  ->where('name', 'like', "%".$search."%")
-                                  ->orWhere('branch', 'like', "%".$search."%");
-                                });
-
-          $filteredEmployees = $search ? $employees->count() : $totalEmployees;
-          $employees = $employees->skip($start)
-                                  ->take($length)
-                                  ->get();
-
-          $response = [
-              'draw' => intval($draw),
-              'recordsTotal' => intval($totalEmployees),
-              'recordsFiltered' => $filteredEmployees,
-              'data' => $employees
-          ];
-          return Response::json($response);
+        //SERVER SIDE RENDERING HANDLE FOR data table
```

Comment:
I noticed you have implemented the `order` query on collection it should works on MySQL not in array.

Reply:
sorry couldn't understand Can you make it more clear? @Raza9798